common/src/main/java/net/william278/husktowns/manager/TownsManager.java

```diff
@@ -689,7 +688,7 @@ public void withdrawMoney(@NotNull OnlineUser user, @NotNull BigDecimal amount)
 
             final Town town = member.town();
             final BigDecimal withdrawal = amount.min(town.getMoney());
-            economy.giveMoney(user, withdrawal);
+            economy.giveMoney(user, withdrawal, "Town "+town.getId()+":"+town.getName()+" withdrawal");
```

Comment:
style: spaces between plus symbols in string concatenation `+`

Reply:
i forgot Ctrl+Alt+L